library/src/main/java/com/novoda/downloadmanager/lib/Query.java

```diff
@@ -12,15 +15,35 @@
  * This class may be used to filter download manager queries.
  */
 public class Query {
+
+    @Retention(RetentionPolicy.SOURCE)
+    @IntDef({ORDER_ASCENDING, ORDER_DESCENDING})
+    public @interface Order {
+    }
```

Comment:
`// i am an Annotation, my body has ascended leaving here only this empty shell`